ethcore/src/spec/spec.rs

```diff
@@ -98,6 +98,8 @@ pub struct CommonParams {
 	pub block_reward: U256,
 	/// Registrar contract address.
 	pub registrar: Address,
+	/// Apply reward
+	pub apply_reward: bool,
```

Comment:
Why is this even needed? Can't we just assume `apply_reward == block_reward != 0`?